baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/PreferSafeLoggingPreconditions.java

```diff
@@ -94,10 +93,12 @@ public Description matchMethodInvocation(MethodInvocationTree tree, VisitorState
             return Description.NO_MATCH;
         }
 
-        SuggestedFix.Builder fix = SuggestedFix.builder();
-        String logSafeQualifiedClassName = SuggestedFixes.qualifyType(state, fix, "com.palantir.logsafe.Preconditions");
+        SuggestedFix.Builder fix = SuggestedFix.builder()
+                .addImport("com.palantir.logsafe.Preconditions")
+                .removeImport("com.google.common.base.Preconditions");
```

Comment:
For our repos `removeImport` should be unnecessary because we run `spotlessApply` inside excavator which actually gets rid of unused imports automatically.